notebooks/02-ingestion/05-working-with-nested-columns.ipynb

```diff
@@ -326,6 +326,44 @@
     "resp.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### New in Druid 29 - Dynamic `path` in JSON_VALUE\n",
+    "JSON_VALUE and JSON_QUERY functions can now use a dynamic `path` parameter by specifying an expression instead of a literal.\n",
+    "\n",
+    "While this can useful it comes with some cost. Values selected in this fashion will not be able to use the optimized columnar access to the data. Using this mode causes Druid to parse the raw JSON value at run-time.\n",
```

Comment:
I think this could be a bit clearer in terms of what you're making the case for here - if you store the object natively, isn't it columnarised? And if you store it in an object, isn't that stored optimally too?

That said advice like this isn't the purpose of the notebooks, so you could remove it. Instead, show examples of both approaches so the reader can decide and they can be referred to from elsewhere.

Reply:
I think it is important to note the change in behavior.
- If you use a literal for the path, it can used columnar access with all the goodness of indexes
- if you use an expression to calculate the path, it cannot use columnar access, so it parses the raw json to achieve this which is way slower.